appveyor.yml

```diff
@@ -0,0 +1,36 @@
+version: 1.19.{build}
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+  - mark/Revit2019
```

Comment:
if this is merged we will have to tweak this file, happy to help with that

Reply:
@ZiyunShang @marchello2000 perhaps we should tweak it now?